src/Runner.Common/JobServerQueue.cs

```diff
@@ -683,6 +807,20 @@ internal class UploadFileInfo
         public bool DeleteSource { get; set; }
     }
 
+    internal class ResultsUploadFileInfo
+    {
+        public Guid TimelineId { get; set; }
+        public Guid TimelineRecordId { get; set; }
```

Comment:
I saw that we're doing the same thing in `UploadFileInfo`: Does anyone know why `TimelineId` and `TimelineRecordId` are Guids while the other IDs we're dealing with are strings here?